content/docs/command-reference/cache/index.md

```diff
@@ -15,11 +15,10 @@ positional arguments:
 
 ## Description
 
-The DVC Cache is where your data files, models, etc. (anything you want to
-version with DVC) are actually stored. The data files and directories visible in
-the <abbr>workspace</abbr> are links\* to (or copies of) the ones in cache.
-Learn more about it's
-[structure](/doc/user-guide/dvc-files-and-directories#structure-of-the-cache-directory).
+The <abbr>DVC cache</abbr> is where your data files, models, etc. (anything you
+want to version with DVC) are actually stored. The data files and directories
+visible in the <abbr>workspace</abbr> are links\* to (or copies of) the ones in
+cache.
```

Comment:
OK, since we moved the concept out of the cmd refs, let's just focus on what the `cache` command (group) is for here please. And link to the concept page

Reply:
> let's just focus on what the cache command (group) is for

p.s. like it was done with `dag` 👍

Reply:
ACTUALLY. We may be able to skip the `cache` index completely and have it jump directly to `cache dir` when you click on this...

Reply:
UPDATE: I've deleted this file and changed the source to `false` in sidebar.json so the nav entry skips to the first child (`dir`). @jeremydesroches please do the same for Parameters once extracted into its concept page.

Reply:
> please do the same for Parameters once extracted into its concept page.

OK, done for Parameters. I also pulled the example from the index there because it was extensive and unique (I think). Good question for the other concepts too: do examples belong or should they remain in cmd ref?

Reply:
> pulled the example from the (params) index there because it was extensive and unique

Good thinking. Always best to err on the side of caution. I'll finalize the concept pages contents when the time comes.

> Good question for the other concepts too: do examples belong or should they remain in cmd ref?

We want to keep examples in cmd refs. Please don't move those (general rule), just conceptual explanations. Thanks